aiida_quantumespresso/parsers/raw_parser_pw.py

```diff
@@ -11,6 +11,8 @@
 import string
 import xml.dom.minidom
 
+import numpy as np
```

Comment:
I don't think this import is used

Reply:
its used to calculate the atom positions, if specified in `crystal` (fractional) coordinates (see line ~1468)